solidity/ecdsa/package.json

```diff
@@ -62,7 +62,7 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
-    "@keep-network/random-beacon": "development",
+    "@keep-network/random-beacon": "../random-beacon",
```

Comment:
Want to leave it as is or set it back to `development` once an updated package is published?

Reply:
Waiting for a new package to be published, I will switch it in this PR.

Reply:
Switched back.